### ISSUE TITLE
Use geocoder API to geocode during ETL

### ISSUE THREAD

**Issue by potash** [Chicago/lead-etl#6]
Currently the ETL process relies on already geocoded data (blood lead tests). That data was geocoded by CDPH. There are two issues:

1) for future refreshes of the data we should not rely on CDPH to do the geocoding
2) for the existing data, the geocoding that CDPH was not great. I reran some of the bad addresses through the geocoder API manually and they were recognized. So we will benefit by doing this for all addresses.

**Comment by tomschenkjr** [comment]
Architecturally, do you think you’ll “bake-in” the code into lead-ETL or create a separate package that is called by lead-etl?

**Comment by potash** [comment]
Yes, I was thinking it would live in this repo. But it would work in such a way that we do not need to re-geocode millions of addresses everytime we run the ETL. What do you think?

**Comment by tomschenkjr** [comment]
Sounds great. Just wanted an idea on the direction.
